Add Alias Helper Method

tyre opened the pull request:
For consistency with the JS API, I added a helper method that mimics `mixpanel.alias` to allow a user to execute something like:

```
Mixpanel.alias('bubbles', {distinct_id: 'walrus'})
```

michaelglass commented:
think it looks good.  for consistency, it should either go in person (fine) or append_alias should also move into event (probably better, I think only things that hit the /engage endpoint should end in person.rb)

tyre commented:
I can move append_alias for consistency. I'll update with that change.

tyre commented:
updated.

michaelglass commented:
:poop: :+1: :+1: :+1: :+1: :poop:

michaelglass commented:
@zevarito MAKE THE MAGIC HAPPEN

zevarito commented:
Hey guys, I am back.

So, haven't much time to look at this, but I think you should add a test and update documentation, then I think it is fine to merge.

tyre commented:
Updated the readme and added some rspec tests. Let me know if there is anything else I can dol :)

zevarito commented:
Hi @tyre,

Sorry I could not use time on the project until now.

Seems like build is failing again right?

tyre commented:
I just ran them again to confirm but I show

```
Finished in 1.05 seconds
74 examples, 0 failures
```

Which ones are failing?

michaelglass commented on spec/mixpanel/tracker_spec.rb:
```diff
@@ -51,6 +51,22 @@
+      end
+
+      it 'includes the aliased name in the properties' do
+        @mixpanel.should_receive(:track_event).with('$create_alias', { alias: 'Cristina Wheatland' }, {}, Mixpanel::Event::TRACK_URL)
```
@tyre this ruby 1.9 hash syntax is causing tests to fail for the ruby 1.8 target  
https://travis-ci.org/zevarito/mixpanel/builds/6891590

tyre committed:
Add alias helper method to event tracking

Also moves Mixpanel::Person#append_alias to Mixpanel::Event
for consistency.

Rspec tests and README update

tyre commented:
Ah zut alors, I have updated that line and Travis is happy again :)

zevarito commented:
Merged, thanks.
